pkg/plugin/backup_pvc_action_plugin.go

```diff
@@ -52,6 +45,12 @@ func (p *NewPVCBackupItemAction) Execute(item runtime.Unstructured, backup *vele
 		return nil, nil, errors.WithStack(err)
 	}
 
+	p.Log.Infof("VSphere PVCBackupItemAction for PVC %s/%s started", pvc.Namespace, pvc.Name)
+	var err error
+	defer func() {
+		p.Log.WithError(err).Infof("VSphere PVCBackupItemAction for PVC %s/%s completed", pvc.Namespace, pvc.Name)
```

Comment:
Why is this an error and why does it needs to be in a defer statement.

Reply:
`defer` makes sure this line of log will always get printed out on the exit of `Execute` func of plugin. `WithError(err)` ensures it will print out errors if any.